db/migrate/20190523225323_enable_unaccent_extension.rb

```diff
@@ -0,0 +1,9 @@
+class EnableUnaccentExtension < ActiveRecord::Migration
+  def up
+    enable_extension "unaccent"
```

Comment:
:see_no_evil: this requires SUPERUSER privileges. I think I'll just do what we did with https://github.com/coopdevs/timeoverflow-provisioning/blob/316c6dfffef7b9a0eafd7737a40eb30c81b8e3c4/roles/database/tasks/main.yml#L56-L61

Reply:
Done in https://github.com/coopdevs/timeoverflow-provisioning/pull/182 already.